css/properties/-webkit-rtl-ordering.json

```
@@ -20,7 +20,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

See: https://github.com/caugner/css-properties/commit/33ce1ea279967afb2272f8b51aa53a147afcb375

```suggestion
              "version_added": "≤4"
```